docs/data-sources/cbr_vaults_by_tags.md

```diff
@@ -0,0 +1,237 @@
+---
+subcategory: "Cloud Backup and Recovery (CBR)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_cbr_vaults_by_tags"
+description: |-
+  Use this data source to get available CBR vaults filtered by tags within Huaweicloud.
+---
+
+# huaweicloud_cbr_tags
+
+Use this data source to get available CBR vaults filtered by tags within Huaweicloud.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_cbr_vaults_by_tags" "test" {
+  action = "filter"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+* `action` - (Required, String) Specifies the action name. Possible values are **count** and **filter**.
+  + **count**: querying count of data filtered by tags.
+  + **filter**: querying details of data filtered by tags.
+
+* `without_any_tag` - (Optional, Bool) Specifies whether ignore tags params.
+  If this parameter is set to true, all resources without tags are queried.
+
+* `tags` - (Optional, List) Specifies the instances that associate with all key/value pairs in this list.
+  The [tags](#tags_struct) structure is documented below.
+
+* `tags_any` - (Optional, List) Specifies the instances that associate with any key/value pairs in this list.
+  The [tags_any](#tags_struct) structure is documented below.
+
+* `not_tags` - (Optional, List) Specifies the instances that associate without all key/value pairs in this list.
+  The [not_tags](#tags_struct) structure is documented below.
+
+* `not_tags_any` - (Optional, List) Specifies the instances that associate without any key/value pairs in this list.
+  The [not_tags_any](#tags_struct) structure is documented below.
+
+* `sys_tags` - (Optional, List) Specifies the instances that associate with enterprise project.
+  The [sys_tags](#tags_struct) structure is documented below.
+
+* `matches` - (Optional, List) Specifies the instances which condition matched.
+  The [matches](#matches_struct) structure is documented below.
+
+* `cloud_type` - (Optional, String) Specifies cloud type of the instances.
+
+* `object_type` - (Optional, String) Specifies resource type of the instances.
+
+<a name="tags_struct"></a>
+The `tags` block supports:
+
+* `key` - (Optional, String) The key of the resource tag.
+
+* `values` - (Optional, List) All values corresponding to the key.
+
+<a name="matches_struct"></a>
+The `matches` block supports:
+
+* `key` - (Optional, String) The key of the resource tag.
```

Comment:
The field descriptions of `key` and `value` are quite different from those in openapi. Please add more information.